op-batcher/batcher/driver.go

```diff
@@ -344,7 +343,43 @@ func (l *BatchSubmitter) publishStateToL1(queue *txmgr.Queue[txData], receiptsCh
 	}
 }
 
-// publishTxToL1 queues a single tx to be published to the L1
+// clearState clears the state of the channel manager
+func (l *BatchSubmitter) clearState(ctx context.Context) {
+	l.Log.Info("Clearing state")
+	defer l.Log.Info("State cleared")
+
+	// Attempt to set the L1 safe origin and clear the state, if fetching fails -- fall through to an infinite retry
+	l1SafeOrigin, err := l.safeL1Origin(ctx)
+	if err != nil {
```

Comment:
```suggestion
	if err == nil {
```
? Also, a log is missing that is present in the loop, see other comment.